modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/ClayCriteriaGroup.es.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import {PropTypes} from 'prop-types';
+import ClayCriteriaRow from './ClayCriteriaRow.es';
+import ClayButton from './ClayButton.es';
+
+function insertAtIndex(item, list, index) {
+	return [...list.slice(0, index), item, ...list.slice(index, list.length)];
+}
+
+class ClayCriteriaGroup extends React.Component {
+	render() {
```

Comment:
The idea was for private methods to go after all lifecycle methods, but in https://issues.liferay.com/browse/LPS-87162 I've moved them back to same order we had in Faro since it made finding the `render` method difficult which is one of the main places I'd go to initially when looking at a component file.